### ISSUE TITLE
Bad behaviour on writing non-active variable output

### ISSUE THREAD

**Issue by nnhjy** [spine-tools/SpineOpt.jl#500]
For variables that are not active in the model, SpineOpt would simply throw an error when writing corresponding outputs:
```
ERROR: LoadError: MethodError: no method matching update_import_data!(::Dict{Symbol, Any}, ::Symbol, ::Dict{Any, Any}; for_object=true, report="main", alternative="")
Closest candidates are:
  update_import_data!(::Dict{Symbol, Any}, ::T, ::Dict{K, V}; for_object, report, alternative) where {T, K<:NamedTuple, V} at C:\Tools\Spine\SpineInterface.jl\src\api.jl:651
Stacktrace:
  [1] write_parameters(parameters::Dict{Any, Any}, url::String; upgrade::Bool, for_object::Bool, report::String, alternative::String, on_conflict::String, comment::String)
    @ SpineInterface C:\Tools\Spine\SpineInterface.jl\src\api.jl:738
``` 
To reproduce the issue:
1. Set up a SpineOpt model that works and is feasible;
2. Link an `output` object term for a variable that would not be used in the model to `report` object in its `report__output` relationship;
3. Run the model and the issue should appear. 

Fortunately, the issue does not apply to objective terms (costs). There needs a user-friendly way to handle those non-active variables.

**Comment by mihlema** [comment]
Could you possible provide a (minimal) working example?

**Comment by nnhjy** [comment]
Sure, this [example.zip](https://github.com/Spine-project/SpineOpt.jl/files/9844304/example.zip) should show the problem.

**Comment by DillonJ** [comment]
Thanks @nnhjy we should fix this as part of 0.7
